Apps365 ltd/short url

cirrussoft opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

sriyen-msft commented:
@cirrussoft **Are you sure you want to delete 1486 files from the Microsoft open-source repo?**

cirrussoft commented:
> 1486

We are not owner of the Pull Request #1486

sriyen-msft commented:
> Yes, please, we have removed those files from our local repository from GitHub

@cirrussoft If you want to remove the files from your repository, you can raise a PR directly into your repository and get it approved and removed by merging into it. But, please let us know the reason to raise this PR to remove all these files in Microsoft's open-source repository?

cirrussoft commented:
> > Yes, please, we have removed those files from our local repository from GitHub
> 
> @cirrussoft If you want to remove the files from your repository, you can raise a PR directly into your repository and get it approved and removed by merging into it. But, please let us know the reason to raise this PR to remove all these files in Microsoft's open-source repository?

No, those are removed from our local repo, not from MS repo

sriyen-msft commented:
> > > Yes, please, we have removed those files from our local repository from GitHub
> > 
> > 
> > @cirrussoft If you want to remove the files from your repository, you can raise a PR directly into your repository and get it approved and removed by merging into it. But, please let us know the reason to raise this PR to remove all these files in Microsoft's open-source repository?
> 
> No, those are removed from our local repo, not from MS repo

@cirrussoft Please make sure that you have created the fork based on the "dev" branch of our [repository](https://github.com/microsoft/PowerPlatformConnectors). When you have a fork, please note that it is a replica of Microsoft's open-source repository, and whatever you make changes into the fork, the changes would be applied to our Microsoft's open-source repo when you raise a PR like this. Please go thru the documentation and let me know if you need any assistance from our team, if required, we can schedule a meeting with you in PST time. Thanks

Amjed-Ayoub commented:
Hi @cirrussoft Can you please remove other connector files from this pull request. Thank you very much

cirrussoft commented:
@Amjed-Ayoub Could you please suggest how can we do the same?
